### PR TITLE
Implement UI for the new social feed

### PR DESCRIPTION
**Pull request by ianopolous**

### CODE REVIEW

**Comment by ianopolous** [comment]
Suggestions:
1. Would be good to have pointer cursor over the filename and thumbnail in the feed.
2. It currently doesn't update, (even after adding the update call)
3. Will need lazy loading of old entries, with a clear barrier at bottom (not an infinite scroll)

**Comment by kevodwyer** [comment]
> Would be good to have pointer cursor over the filename and thumbnail in the feed.
Done
> Will need lazy loading of old entries, with a clear barrier at bottom (not an infinite scroll), along with starting at the last seen index rather than 0.
Paging done. Display most recent at top. Page size 20. Can get addtional by scrolling down until reaching end of viewing area. Temrinal message displayed when no more entries to display.
> Set the last seen index.
Done
>Currently items are not displayed in the order social feed returns them
See paging above

File: src/components/filesystem/filesystem.html [hunk 1]
```diff
@@ -7,8 +7,11 @@
...
 	<li id="uploadButton" @click="toggleUploadMenu()" v-if="isWritable" class="navbar-brand nopad"><a><span data-toggle="tooltip" data-placement="bottom" title="Upload" class="pnavbar fa fa-upload" style="cursor: pointer;"/></a></li>
 	<li id="mkdirButton" @click="askMkdir()" v-if="isWritable" class="navbar-brand nopad"><a><span data-toggle="tooltip" data-placement="bottom" title="Create new directory" class="pnavbar fas fa-folder-plus" style="cursor: pointer;"/></a></li>
-	<li id="alternateViewButton" @click="switchView()" class="navbar-brand nopad"><a><span data-toggle="tooltip" data-placement="bottom" title="Switch view" id="altViewSpan" v-bind:class="['pnavbar', 'fa', 'tour-view', grid ? 'fa-list' : 'fa-th-large']" style="cursor: pointer;"/></a></li>
-	<li id="userOptionsButton" v-if="isLoggedIn" @click="showSocialView()" class="navbar-brand nopad">
+        <li id="alternateViewButton" @click="switchView()" class="navbar-brand nopad"><a><span data-toggle="tooltip" data-placement="bottom" title="Switch view" id="altViewSpan" v-bind:class="['pnavbar', 'fa', 'tour-view', grid ? 'fa-list' : 'fa-th-large']" style="cursor: pointer;"/></a></li>
```
Comment (ianopolous) [inline src/components/filesystem/filesystem.html:10]: why has indentation increased?

**Review by kevodwyer — APPROVED** [review]

**Review by kevodwyer — APPROVED** [review]

File: src/components/filesystem/filesystem.html (hunk 1 above)
Comment (kevodwyer) [reply src/components/filesystem/filesystem.html]: fixed

**Pull request merged**